### DISCUSSION TITLE
Line wrapping in bullet list?

### DISCUSSION THREAD

**Discussion by boldandbusted** [maaslalani/slides#46] [Q&A] [answered]
Howdy. In my tmux'd terminal, when I have bullet lists, the text is not line-wrapped, and disappears off the edge. I can see that slides is finding the dimensions of the terminal screen because the pager ("Slide X / Y") is still within the terminal screen bounds. Is there a setting I'm missing to make line-wrapping occur? Thanks! :)

**Comment by maaslalani** [comment]
I can't remember exactly why but I turned off line wrapping for the bullet points https://github.com/maaslalani/slides/blob/cd4053bb692a0d2447d56e97ee6fe2f60d7e8906/internal/model/model.go#L136.

I think the idea was that the presenter should be responsible for adding new lines and formatting their presentation otherwise it might get annoying if we pick an arbitrary number.

**Reply by maaslalani** [reply]
We can probably make this configurable (please create an issue if you want this)  but I would highly suggest simply adding new lines

**Comment by maaslalani** [comment, marked as answer]
Btw, this is now fixed to wrap to your terminal width https://github.com/maaslalani/slides/pull/84
